### DISCUSSION TITLE
Can't access vercel.com anymore: ERR_CONNECTION_CLOSED

### DISCUSSION THREAD

**Discussion by pascalschwahn** [vercel/community#6305] [Account Issue] [answered]
### Question

Since yesterday evening I can no longer access vercel.com in my home network. This happened after I accidentally deleted the GitHub connection. I was able to re-establish the connection between GitHub and vercel via the mobile network. On my home network, however, I always get the error "ERR_CONNECTION_CLOSED" when I try to open vercel.com.

**Comment by ibrahim4006** [comment]
i am troubling same error. Did you find any solution

**Comment by amyegan** [comment]
Do you see the same error if you try to access [vercel.com](https://vercel.com) from an incognito window or a different browser?

**Reply by pascalschwahn** [reply]
Yes i see the same error. Also on every other device that is within my home network.

**Reply by pascalschwahn** [reply, marked as answer]
I have just tried it again and now everything is working in my home network. Apparently my IP was blocked for a certain period of time.
